visualization/app/codeCharta/ui/filePanel/filePanelFileSelector/filePanelFileSelector.component.html

```diff
@@ -15,6 +15,10 @@
 	<mat-option *ngFor="let file of files" [value]="file.file.fileMeta.fileName">
 		{{ file.file.fileMeta.fileName }}
 
-		<cc-remove-file-button [filename]="file.file.fileMeta.fileName" [onRemoveFile]="handleOnRemoveFile"></cc-remove-file-button>
+		<cc-remove-file-button
+			[filename]="file.file.fileMeta.fileName"
+			[onRemoveFile]="handleOnRemoveFile"
+			[files]="files"
```

Comment:
we could test the number of files here *ngIf="files.length > 1", thus the delete button wouldn't apear